lib/policy_machine/association.rb

```diff
@@ -35,34 +24,15 @@ def initialize(stored_user_attribute, stored_set_of_operation_objects, stored_op
       )
     end
 
-    # Returns true if the operation set of this association includes the given operation.
-    #
-    def includes_operation?(operation)
-      # Note:  set_of_operation_objects.member? isn't calling PM::PolicyElement ==
-      set_of_operation_objects.any? { |op| op == operation }
-    end
-
     # Create an association given persisted policy elements
     #
-    def self.create(user_attribute_pe, set_of_operation_objects, operation_set, object_attribute_pe, policy_machine_uuid, pm_storage_adapter)
+    def self.create(user_attribute_pe, operation_set, object_attribute_pe, policy_machine_uuid, pm_storage_adapter)
       # argument errors for user_attribute_pe
       raise(ArgumentError, "user_attribute_pe must be a UserAttribute.") unless user_attribute_pe.is_a?(PM::UserAttribute)
       unless user_attribute_pe.policy_machine_uuid == policy_machine_uuid
         raise(ArgumentError, "user_attribute_pe must be in policy machine with uuid #{policy_machine_uuid}")
       end
 
-      # argument errors for set_of_operation_objects
-      raise(ArgumentError, "set_of_operation_objects must be a Set of Operations") unless set_of_operation_objects.is_a?(Set)
-      raise(ArgumentError, "set_of_operation_objects must not be empty") if set_of_operation_objects.empty?
-      set_of_operation_objects.each do |op|
-        unless op.is_a?(PM::Operation)
```

Comment:
You just mean something that checks that the operation set is an operation set? Yea, that's directly below this...you added it 😝 .